cli/sync.ts

```diff
@@ -1,67 +1,69 @@
-import * as flags from "../deps/std/flags.ts"
+import { Command } from "../deps/cliffy.ts"
 import { blue, gray } from "../deps/std/fmt/colors.ts"
 import { assertEquals } from "../deps/std/testing/asserts.ts"
 import { syncNets } from "../server/mod.ts"
 import { normalizePackageName } from "../util/mod.ts"
 import { tempDir } from "../util/tempDir.ts"
 import { resolveNets } from "./resolveNets.ts"
 
-export default async function(...args: string[]) {
-  const {
-    "import-map": importMapFile,
-    "package-json": packageJsonFile,
-    check,
-    out,
-    server,
-  } = flags.parse(args, {
-    string: ["config", "import-map", "out", "package-json", "server"],
-    boolean: ["check"],
-    default: {
-      server: "https://capi.dev/",
-      out: "target/capi",
-    },
+export default new Command()
+  .description("Syncs metadata and codegen")
+  .option("-n, --nets <nets:file>", "nets.ts file path", { default: "./nets.ts" })
+  .option("--import-map <import-map:file>", "import_map.json file path")
```

Comment:
Can we break these apart into subcommands? Ie. `sync node` and `sync deno`.

Reply:
@harrysolovay we won't need the `--import-map` or `--package-json` options, right?

If it's `deno` or `node` these files can be inferred.